dev-docs/bidders/redtram.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: bidder
+title: Redtram
+description: Prebid Redtram Bidder Adapter.
+pbjs: true
+biddercode: redtram
+media_types: banner
```

Comment:
You're missing a lot of metadata attributes here. Please check the documentation at https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter -- Prebid publishers need these flags to understand your adapter's capabilities.